members/admin/__init__.py

```diff
@@ -283,7 +288,7 @@ def queryset(self, request, queryset):
 
 class ActivityParticipantListCurrentYearFilter(admin.SimpleListFilter):
     # Title shown in filter view
-    title = "Efter aktivitet (år " + str(timezone.now().year) + ")"
+    title = "Efter aktivitet (\u2265 år " + str(timezone.now().year) + ")"
```

Comment:
En kommentar ovenover, der siger hvad der står her ville være lækkert, da jeg i hvert fald ikke i hovedet kan huske hvad u2264 er :slightly_smiling_face:

Reply:
ok er rettet
    # Title shown in filter view. \u2265 : ≥ (større end eller lig med)